### ISSUE TITLE
vcprompt explodes in newly-inited git repository

### ISSUE THREAD

**Issue by bjeanes** [djl/vcprompt#5]
To see this problem simply do the following:
1. `mkdir vctest`
2. `cd vctest`
3. `git init`
4. If vcprompt is in your prompt you will already see the problem, else just run `vcprompt`

Here's the error:

```
~/S/P/test ♪ vcprompt
Traceback (most recent call last):
  File "/Users/bjeanes/.config/misc/vcprompt.py", line 676, in <module>
    print main()
  File "/Users/bjeanes/.config/misc/vcprompt.py", line 298, in main
    systems=options.systems, unknown=options.unknown)
  File "/Users/bjeanes/.config/misc/vcprompt.py", line 234, in vcprompt
    prompt = vcs(path, format, unknown)
  File "/Users/bjeanes/.config/misc/vcprompt.py", line 524, in git
    with open(hash_file, 'r') as f:
IOError: [Errno 2] No such file or directory: '/Users/bjeanes/Sites/Personal/test/.git/refs/heads/master'
```

**Comment by djl** [comment]
Oh, nice catch!

Fixed in 4e9d61ed69d28e19d5e0bc0b9fcedaa2d9b5eb6d. The problem was limited to only `%h` and `%r` format args.

**Comment by bjeanes** [comment]
Yup, that did the trick!
